content/blog/tune-1-2-0/index.Rmd

```diff
@@ -0,0 +1,183 @@
+* Once foreach is fully deprecated, users will be able to use the [interactive logger](https://www.tidyverse.org/blog/2023/04/tuning-delights/#interactive-issue-logging) when tuning in parallel.
+
+From our perspective, transitioning our parallelism support to future makes our packages much more maintainable, reducing complexity in random number generation, error handling, and progress reporting.
+
+In the next release of the package, you'll see a deprecation warning when a foreach parallel backend is registered but no future plan has been specified, so start transitioning your code sooner than later!
+
+## Percentile confidence intervals
+
+Following up on changes in the [most recent rsample release](https://github.com/tidymodels/rsample/releases/tag/v1.2.0), tune introduced a [method for `int_pctl()`](https://tune.tidymodels.org/dev/reference/int_pctl.tune_results.html) that calculates percentile confidence intervals for performance metrics. To calculate a 90% confidence interval for the values of each performance metric returned in `collect_metrics()`, we'd write:
+
+```{r}
+#| label: int-pctl
+set.seed(2024)
+
+int_pctl(xgb_res, alpha = .1)
+```
+
+Note that the output has the same number of rows as the `collect_metrics()` output: one for each unique pair of metric and workflow.
+
+This is very helpful for validation sets. Other resampling methods generate replicated performance statistics. We can compute simple interval estimates using the mean and standard error for those. Validation sets produce only one estimate, and these bootstrap methods are probably the best option for obtaining interval estimates. 
+
+## Relocation of ellipses
+
+We've made a **breaking change** in argument order for several functions in the package (and downstream packages like finetune and workflowsets). Ellipses (...) are now used consistently in the package to require optional arguments to be named. For functions that previously had unused ellipses at the end of the function signature, they have been moved to follow the last argument without a default value, and several other functions that previously did not have ellipses in their signatures gained them. This applies to methods for `augment()`, `collect_predictions()`, `collect_metrics()`, `select_best()`, `show_best()`, and `conf_mat_resampled()`.
+
+## Compute new metrics without re-fitting
+
+We also added a new function, [`compute_metrics()`](https://tune.tidymodels.org/dev/reference/compute_metrics.html), that allows for calculating metric values for metrics that were not used when evaluating against resamples. For example, consider our `xgb_res` object. Since we didn't supply any metrics to evaluate, and this model is a regression model, tidymodels selected Root Mean Squared Error and R-Squared as defaults:
```

Comment:
```suggestion
We've also added a new function, [`compute_metrics()`](https://tune.tidymodels.org/dev/reference/compute_metrics.html), that allows for calculating metrics that were not used when evaluating against resamples. For example, consider our `xgb_res` object. Since we didn't supply any metrics to evaluate, and this model is a regression model, tidymodels selected Root Mean Squared Error and R-Squared as defaults:
```